doc/main.dox

```diff
@@ -0,0 +1,15 @@
+namespace lsst { namespace meas { namespace extensions { namespace photometryKron
+
+/**
+\mainpage lsst::meas::extensions::photometryKron: Kron Photometry
+
+\section photometryKron_introduction Introduction
+
+Kron Photometry Measurement Extension.
+
+This package provides an \ref measBaseIntroduction "lsst.meas.base" compliant
```

Comment:
I'm not about to correct your English, but shouldn't it be "a" here :)
Otherwise, looks fine, if somewhat minimal.